build/meson/README.md

```diff
@@ -7,7 +7,7 @@ modern software development tools and practices, such as unit tests,
 coverage reports, Valgrind, CCache and the like.
 
 This Meson build system is provided with no guarantee and maintained
-by Dima Krasner <dima@dimakrasner.com>.
+by Dima Krasner \<dima@dimakrasner.com\>.
```

Comment:
@lzutao , should you wish it, maybe you could add your name, considering the very large modifications you contributed to this build system